rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/TestLaunchInformation.xtend

```diff
@@ -17,8 +17,10 @@ import java.util.Map
 import org.eclipse.core.resources.IProject
 import org.eclipse.xtend.lib.annotations.Accessors
 import org.testeditor.rcp4.tcltestrun.TclLauncher
+import org.eclipse.xtend.lib.annotations.Data
 
-class LastTestLaunchInformation {
+@Data
+class TestLaunchInformation {
 
 	@Accessors
```

Comment:
sorry, didn't mention that @Accessors are not needed anymore (if annotated with @Data)